sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareProtocols.java

```diff
@@ -9,8 +9,8 @@
 /**
  * Represents protocols that can be set on a share.
  */
-public class ShareEnabledProtocols {
-    private final ClientLogger logger = new ClientLogger(ShareEnabledProtocols.class);
+public class ShareProtocols {
+    private final ClientLogger logger = new ClientLogger(ShareProtocols.class);
 
     private boolean smb;
```

Comment:
Should this be smbEnabled? Just to follow true Fluent?

Reply:
Fixed